docs/en_US/NAS/NasVisualization.rst

```diff
@@ -0,0 +1,4 @@
+NAS Visualization
+=================
+
+Will be supported soon.
```

Comment:
add the NAS Visualization issue here so that others knows we are working on it (in which iteration).